repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -191,8 +192,62 @@ export const getLinesList = (
     return linesList;
 };
 
+export const getExemptedLinesList = (
```

Comment:
If this function is only called for exempted lines, could we not just pass the function the array of exemptedServices instead of the whole ticket?